apps/chat/src/components/Chat/ConversationSettings.tsx

```diff
@@ -70,13 +70,16 @@ export const ModelSelectRow = ({ item, isNotAllowed }: ModelSelectRowProps) => {
   return (
     <div
       className={classNames(
-        'flex items-center gap-2',
+        'flex items-center gap-2 truncate',
         isNotAllowed && 'text-secondary',
       )}
     >
       <ModelIcon entity={item} entityId={item.id} size={18} />
-      <div>
-        <span>{getOpenAIEntityFullName(item)}</span>
+      <div className="truncate">
+        <span>
+          {getOpenAIEntityFullName(item)}{' '}
+          <span className="text-secondary">{item.version}</span>
```

Comment:
show version only if exists and marging (`mr-2`?) instead of
```suggestion
          {getOpenAIEntityFullName(item)}
          {item.version && <span className="text-secondary mr-2">{item.version}</span>}
```
Should we show any prefix `v.`?